testdata/fuzz/Fuzz.t.sol

```diff
@@ -18,7 +18,7 @@ contract FuzzTest is DSTest {
 
     function testFailFuzz(uint8 x) public {
         emit log("testFailFuzz");
-        require(x == 5, "should revert");
+        require(x > 128, "should revert");
```

Comment:
btw I also tried to log the value here via console.log (import "../logs/console.log")

but can't remember if it printed logs on panic or not

Reply:
I just ran the test locally a few times and it did "fail to fail" sometimes. Logged out the value in the strategy itself and did also confirm that it did not pass `5` sometimes. It's a bit frustrating because the strategies we have really only come into effect once a failure has been found... Each test case generates a new tree, effectively a new random value, without respect for what has been tried before

Reply:
ah ok that makes sense, so we reduce 1/256 1/2

sgtm